packages/app/src/components/nominationBanner/declineAppBtn.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const DeclineAppBtn = (props) => {
+
+  return ( 
+      <button
+        disabled={props.status == 'Declined'}
```

Comment:
Not a big deal in this case but it's generally best practice to use a triple equals (`===`) because if you don't you can end up with weirdness like `"1" == 1` being true.

Reply:
that's funny, that's what john and i said!  But when I refactored I didn't want to change other peoples code only alter the code minimally to create 4 new componets

Reply:
will update now of course

Reply:
Yeah that makes sense. In this case it's a pretty harmless change though so you should be fine.